managed/services/telemetry/config.default.yml

```
@@ -951,12 +948,19 @@ telemetry:
       - metric_name: "postgresql_db_count"
         value: 1
 
-  - id: PMMServerHAEnabled
+  - id: PMMServerTelementryEnabled
```

If we receive any metric then telemetry is enabled, isn't it?
```suggestion
  - id: PMMServerTelemetryEnabled
```